packages/salesforcedx-utils-vscode/test/unit/telemetry/telemetry.test.ts

```diff
@@ -279,14 +273,39 @@ describe('Telemetry production mode', () => {
     const telemetryEnabled = await telemetryService.isTelemetryEnabled();
     expect(telemetryEnabled).to.be.eql(false);
 
-    await telemetryService.sendCommandEvent('create_apex_class_command', [
-      0,
-      123
-    ]);
+    telemetryService.sendCommandEvent('create_apex_class_command', [0, 123]);
     assert.notCalled(reporter);
     expect(teleStub.firstCall.args).to.eql([false]);
   });
 
+  xit('Should show telemetry info message', async () => {
+    // create vscode extensionContext in which telemetry msg has never been previously shown
```

Comment:
do these tests need to be skipped?

Reply:
They have the same issue as the tests above them https://github.com/forcedotcom/salesforcedx-vscode/pull/2655/files#r510284491

Reply:
Ah okay gotcha, maybe we can leave comment with that info